src/components/header/styles.js

```diff
@@ -0,0 +1,44 @@
+import styled, { css } from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const HeaderContainer = styled.header`
+  height: 100px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+`;
+
+export const Nav = styled.nav`
+  width: calc(100% - 160px);
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+`;
+
+export const LogoLink = styled(Link)`
+  cursor: pointer;
+`;
+
+export const MenuList = styled.ul`
```

Comment:
I would move the menu styles necxt to the Menu component. If you several components you can name the corresponding style files for example Menu.style.js